Documentation/User/Index.rst

```diff
@@ -197,3 +196,119 @@ collections or even to reindex all documents on the given page.::
        Show each processed documents uid and location with timestamp and
        amount of processed/all documents.
    :Example:
+
```

Comment:
Thank you.

Please add the "--owner" command to the kitodo:index and kitodo:reindex command descriptions, too.

Reply:
Ah, sorry. I missed that!